addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,11 +10976,11 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot for unwatched items (Hides plot and thumb when disabled)"
```

Comment:
keep it short as it's likely to scroll if it's that long, so phil's version is IMO better